workers/blescanmulti.py

```diff
@@ -23,12 +23,14 @@ def searchmac(self, devices, mac):
 
     return None
 
-  def status_update(self):
+  def status_update(self, poll_device=None):
```

Comment:
Maybe it should be generalized to something like: device_filter=None , which will act as filter for self.devices, then that if and continue wont be needed inside each loop. When self.devices will be a getter method ( see other comment ) it could take array to filter devices and return only filtered subset.